runtime/interpreter/errors.go

```diff
@@ -43,6 +46,8 @@ func (e *unsupportedOperation) Error() string {
 	)
 }
 
+func (e *unsupportedOperation) IsUserError() {}
```

Comment:
This is actually an internal/implementation error, it is e.g. thrown when a program that type-checked could not be interpreted, e.g. currently missing support for certain binary and unary operators.

Reply:
Fixed it in https://github.com/onflow/cadence/pull/1718/commits/ff5e6de74cf4a640f71b6e91f882b25952966e46